homeworks/src/homework11/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,49 @@
+package homework11.coffee.order;
+
+import java.util.Comparator;
+import java.util.PriorityQueue;
+import java.util.Queue;
+
+public class CoffeeOrderBoard {
+    private Queue<Order> orderBoard;
+    private static int counter = 1;
+
+    public CoffeeOrderBoard() {
+        orderBoard = new PriorityQueue<>(5, (o1, o2) -> o1.getOrderNumber() - o2.getOrderNumber());
```

Comment:
You also may use another queue, for example - LinkedList. Because we adding a new order at the end of the list. Up to you